refactor/urls.js

```diff
@@ -1,16 +1,19 @@
-// Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+    if (tasks?.prevTasks) {
+        return { url: '/tasks?hasPrev=true' };
+    }
+    
+    let url = '/tasks'; 
+    let params = dev ? { status: "ACTIVE", size: 20, dev } : {};
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
+    if (tasks?.nextTasks) {
+        params = { ...params, hasNext: true }
     }
 
-    if (tasks.prevTasks) {
-        url = '/tasks?hasPrev=true';
+    if (Object.keys(params).length > 0) {
```

Comment:
Do we need the 'if'?

Reply:
Not necessarily, It is there to avoid the trailing `?` after the url.
If I had skip `if` condition, the url might have been correct but with a `?`

It may not make any difference though to the url if hit. 
It would've been acceptable had I skipped if ?

@ankushdharkar